python/ctsm/test/test_sys_fsurdat_modifier.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+"""System tests for fsurdat_modifier
+
+"""
+
+import os
+
+import unittest
+import tempfile
+import shutil
+from subprocess import call
+from configparser import ConfigParser
+
+import xarray as xr
+
+import ctsm.unit_testing
+from ctsm.utils import get_config_value
+
+
+class TestSysFsurdatModifier(unittest.TestCase):
+    """System tests for fsurdat_modifier"""
+
+    def setUp(self):
+        self._tempdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self._tempdir, ignore_errors=True)
+
+    def test_minimalInfo(self):
+        """
+        This test specifies a minimal amount of information
+        """
+
+        # get fsurdat_in and fsurdat_out from the .cfg (config) file
+        cfg_path = 'ctsm/test/testinputs/modify_minimal.cfg'
```

Comment:
@billsacks I created directory /ctsm/test/testinputs, as I understood you to recommend. As of now it contains two configure files and an fsurdat_in file:
- modify_minimal.cfg
- modify_all.cfg
- surfdata_5x5_amazon_16pfts_Irrig_CMIP6_simyr2000_c171214.nc

Would you have me add/commit/push these to the PR or would I eventually rimport them?